SocialCareCaseViewerApi/V1/UseCase/CaseRecordsUseCase.cs

```diff
@@ -23,14 +22,25 @@ public class CaseRecordsUseCase : ICaseRecordsUseCase
         private readonly IDatabaseGateway _databaseGateway;
         private readonly IMongoGateway _mongoGateway;
 
-        public CaseRecordsUseCase(IProcessDataGateway processDataGateway, IDatabaseGateway databaseGateway, IMongoGateway mongoGateway)
+        public CaseRecordsUseCase(IProcessDataGateway processDataGateway, IDatabaseGateway databaseGateway,
+            IMongoGateway mongoGateway)
         {
             _processDataGateway = processDataGateway;
             _databaseGateway = databaseGateway;
             _mongoGateway = mongoGateway;
         }
+
         public CareCaseDataList GetResidentCases(ListCasesRequest request)
         {
+            IEnumerable<string> caseExclusionList = new List<string>
```

Comment:
this could perhaps be defined in the if statement below since it's only ever needed if the flag is true